node/services/config.go

```diff
@@ -0,0 +1,44 @@
+package services
+
+import (
+	"encoding/hex"
+
+	"github.com/libp2p/go-libp2p-core/peer"
+	"github.com/multiformats/go-multiaddr"
+	tmbytes "github.com/tendermint/tendermint/libs/bytes"
+)
+
+type Config struct {
+	// GenesisHash is the Block/Header hash that Nodes use as starting point for header synchronization.
+	// Only affects the node once on initial sync.
+	GenesisHash string
+	// TrustedPeer is the peer we trust to fetch headers from.
+	// Note: The trusted does *not* imply Headers are not verified, but trusted as reliable to fetch headers
+	// at any moment.
+	TrustedPeer string
+}
+
+// TODO(@Wondertan): We need to hardcode genesis hash and one bootstrap peer as trusted.
```

Comment:
technically we don't, just have to make sure they're passed in as they're gonna change dependent on devnet set-up.